Guardfile

```diff
@@ -0,0 +1,37 @@
+# A sample Guardfile
+# More info at https://github.com/guard/guard#readme
+
+## Uncomment and set this to only include directories you want to watch
+# directories %w(app lib config test spec features) \
+#  .select{|d| Dir.exists?(d) ? d : UI.warning("Directory #{d} does not exist")}
+
+## Note: if you are using the `directories` clause above and you are not
+## watching the project directory ('.'), then you will want to move
+## the Guardfile to a watched dir and symlink it back, e.g.
+#
+#  $ mkdir config
+#  $ mv Guardfile config/
+#  $ ln -s config/Guardfile .
+#
+# and, you'll have to watch "config/Guardfile" instead of "Guardfile"
+
+guard :bundler do
+  require 'guard/bundler'
+  require 'guard/bundler/verify'
+  helper = Guard::Bundler::Verify.new
+
+  files = ['Gemfile']
+  files += Dir['*.gemspec'] if files.any? { |f| helper.uses_gemspec?(f) }
+
+  # Assume files are symlinked from somewhere
+  files.each { |file| watch(helper.real_path(file)) }
+end
+
+guard :rspec, all_on_start: true, all_after_pass: true,
```

Comment:
You are saying to take out: cmd: 'bundle exec rspec' do
?

Am I understanding correctly?

Laurie
> On Apr 7, 2016, at 1:59 PM, Jason Noble <notifications@github.com> wrote:
> 
> In Guardfile <https://github.com/LaurieLinz/hyperbaric_medicine/pull/1#discussion_r58934808>:
> 
> > +#
> > +# and, you'll have to watch "config/Guardfile" instead of "Guardfile"
> > +
> > +guard :bundler do
> > +  require 'guard/bundler'
> > +  require 'guard/bundler/verify'
> > +  helper = Guard::Bundler::Verify.new
> > +
> > +  files = ['Gemfile']
> > +  files += Dir['*.gemspec'] if files.any? { |f| helper.uses_gemspec?(f) }
> > +
> > +  # Assume files are symlinked from somewhere
> > +  files.each { |file| watch(helper.real_path(file)) }
> > +end
> > +
> > +guard :rspec, all_on_start: true, all_after_pass: true,
> 🎉
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/LaurieLinz/hyperbaric_medicine/pull/1/files/be5fba3da83e1881ac1a866a34da0b2a4f106216#r58934808>